### PR TITLE
Fix Pysam recipe 0.16.0 with jmarshalls fixes

### PR DESCRIPTION
**Pull request by apeltzer**
<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/pysam/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   version: '{{ version }}'
 
 source:
   url: https://github.com/pysam-developers/pysam/archive/v{{ version }}.tar.gz
-  sha256: 72f01e2249d2b552264f2d932f548d48e84f61e017504a7aa623aba51e656e75
+  sha256: 55a0ecc681ef1bbe847c658135ceb918f03a0d93aafb4d48b9cda0b098c1d4df
 
 build:
-  number: 1
+  number: 0
```
Comment (apeltzer) [inline recipes/pysam/meta.yaml:12]:
```suggestion
  number: 1
```

File: recipes/pysam/meta.yaml [hunk 2]
```diff
@@ -23,17 +23,16 @@ requirements:
...
     - xz
     - bzip2
     - libdeflate
-    - curl
-
-test:
-  imports:
-    - pysam
+    - libcurl
+    - openssl  # [not osx]
```
Comment (apeltzer) [inline recipes/pysam/meta.yaml:36]:
```suggestion
    command:
        python run_test.py
```

Comment (apeltzer) [reply recipes/pysam/meta.yaml:36]: Not sure whether one can actually include a test like this in such a PR @bioconda/core might know about this.

**Comment by jmarshall** [comment]
@apeltzer: Just. Stop.

**Pull request closed without merging**